src/com/geekhub/hw2/vehicle/SolarCar.java

```diff
@@ -0,0 +1,28 @@
+package com.geekhub.hw2.vehicle;
+
+import com.geekhub.hw2.parts.EnergySource;
+import com.geekhub.hw2.parts.Engine;
+import com.geekhub.hw2.parts.Gear;
+
+public class SolarCar extends Car {
+    public SolarCar(EnergySource energySource, Engine engine, Gear gear) {
```

Comment:
One of the solutions could be next: make constructor without parameters and pass to the super contructor default implementations.